src/Amplitude.js

```diff
@@ -1,3 +1,10 @@
+/*
```

Comment:
Looks like we need to use double-stars at the top to make this a doc comment instead of a regular multiline comment:

```suggestion
/**
```